Cars/cars_2.py

```diff
@@ -0,0 +1,118 @@
+import random
+
+
+class Car:
+    def __init__(self, brand, model, color, economy):
+        self.mileage = 0
+        self.fuel = 100
+        self.brand = brand
+        self.model = model
+        self.color = color
+        self.economy = economy
+        self.fuel_consumed = 0
+
+    def drive(self, distance):
+        fuel_needed = distance / 100 * self.economy
+        if fuel_needed <= self.fuel:
+            self.mileage += distance
+            self.fuel -= fuel_needed
+            self.fuel_consumed += fuel_needed
+            print(f"Автомобіль {self.brand} {self.model} ({self.color}) проїхав {distance} км.")
+        else:
+            print("Недостатньо палива для завершення поїздки.")
+
+    def distance_left(self):
```

Comment:
also for methods with `return` - it is also good to show the return type. `def distance_left(self): -> float`